packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -2937,12 +2941,10 @@ class _RouteEntry extends RouteTransitionRecord {
   /// remove as a result of a page update.
   static const int kDebugPopAttemptLimit = 100;
 
-  static final Route<dynamic> notAnnounced = _NotAnnounced();
```

Comment:
nit: IMO keeping this static field around makes the code below a little more readable and matches the pattern Dart/Flutter uses elsewhere for these kind of consts (e.g. Duration.zero, Size.zero, etc)